plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/FileSystemExchangeSourceHandle.java

```diff
@@ -57,6 +57,14 @@ public int getPartitionId()
         return partitionId;
     }
 
+    @Override
+    public long getDataSizeInBytes()
```

Comment:
nit: worth caching?

Reply:
(preexistent)

Reply:
I fixed it to be called only once per handle. Caching is not ideal as it would use an extra memory. Ideally it shouldn't be called more than once.